src/game/server/neo/neo_player.cpp

```diff
@@ -1268,16 +1268,16 @@ float CNEO_Player::GetFogObscuredRatio(CBaseEntity* target) const
 	}
 
 	// Injured Target Impact
-	detectionChance *= (1.0f + ((1 - targetPlayer->HealthFraction()) * MULT_INJURED_PER_HEALTH_POINT_FACTOR));
+	detectionChance *= 1.0f + (MULT_BLEEDING_INJURY_EVENT_FACTOR * m_iBotDetectableBleedingInjuryEvents);
 
 	// Assault class motion vision
 	if (GetClass() == NEO_CLASS_ASSAULT && targetIsMoving)
 	{
-		detectionChance = MAX(detectionChance, MIN_ASSAULT_DETECTION_CHANCE_PER_TICK);
+		detectionChance = Max(detectionChance, MIN_ASSAULT_DETECTION_CHANCE_PER_TICK);
 	}
 
 	// Ensure the final detection chance is within valid bounds [0, 1] (as a ratio)
-	detectionChance = MAX(0.0f, fminf(1.0f, detectionChance));
+	detectionChance = Max(0.0f, fminf(1.0f, detectionChance));
```

Comment:
Sorry to keep harping on this, but could you also change the `fminf` -> `Min` on this line please.

Also on second look, since this is essentially a clamp, it might be neater to use: https://github.com/NeotokyoRebuild/neo/blob/ddaca40a07ea837f97c41570f6f5994d63879bb4/src/public/tier0/basetypes.h#L93-L106 rather than doing the Max(Min(...)) pattern? `detectionChance = Clamp(detectionChance, 0.f, 1.f);` or so. But whichever is fine.